UI/window-basic-preview.hpp

```diff
@@ -42,6 +43,7 @@ class OBSBasicPreview : public OBSQTDisplay {
 
 	friend class SourceTree;
 	friend class SourceTreeItem;
+	friend class OBSBasic;
```

Comment:
I won't fight this if it's difficult to fix, but I find adding `friend class` everywhere a bad habit of us.

Reply:
Fixed